app/main/store.ts

```diff
@@ -21,13 +21,17 @@ const useStore = create<Store>((set) => ({
       title: "[블루아카이브] 1.5주년 신규 OST [Thanks to] with 윤하",
       thumbnail: "https://i.ytimg.com/vi/qpBkMvEyNjc/hq720.jpg",
       alt: "[블루아카이브] 1.5주년 신규 OST [Thanks to] with 윤하",
+      artist: "블루 아카이브",
+      tags: ["#추천게임동영상", "#추천동영상"],
     },
     {
       src: "https://www.youtube.com/embed/VzzvtfS4_aE",
       title:
         "블루 아카이브 호시노 콧노래 리믹스 / Blue Archive Hoshino Song (Quilt remix)",
       thumbnail: "https://i.ytimg.com/vi/VzzvtfS4_aE/hq720.jpg",
       alt: "블루 아카이브 호시노 콧노래 리믹스 / Blue Archive Hoshino Song (Quilt remix)",
+      artist: "quilt heron",
+      tags: ["#추천게임동영상"],
```

Comment:
```suggestion
      tags: ["추천게임동영상"],
```
해시가 값에 반드시 붙어야 할까용 ? 태그 벨류만 들고있고 나머지는 좀 더 지유롭게 핸들링 할 수 있도록 하는건 어떨끼용 ?

Reply:
알겠습니다!